enterprise-suite/templates/alertmanager.yaml

```diff
@@ -31,6 +31,10 @@ spec:
               cpu: {{ default .Values.defaultCPURequest .Values.alertManagerCPURequest }}
               memory: {{ default .Values.defaultMemoryRequest .Values.alertManagerMemoryRequest }}
           env:
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
           args:
             - --config.file=/etc/config/alertmanager.yml
             - --storage.path=/data
```

Comment:
Also need to pass in the alertmanager argument here with the podIP.

Reply:
sorry, don't understand what you mean by this

Reply:
oh, i get it - i'm an idiot and haven't actually used the IP in the container `--cluster.listen-address=$(POD_IP):6783`